wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/WatchListScreen.kt

```diff
@@ -117,7 +129,7 @@ fun WatchListScreen(
     }
 }
 
-@Preview(device = WearDevices.SMALL_ROUND, showSystemUi = true)
```

Comment:
`showSystemUi` was showing the Pocket Casts icon in the middle of the screen. It might be an Android Studio issue but I have removed it for now.